src/Features/Core/Portable/InlineParameterNameHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.InlineParameterNameHints
+{
+    internal abstract class AbstractInlineParameterNameHintsService : IInlineParameterNameHintsService
+    {
+        public async Task<IEnumerable<InlineParameterHint>> GetInlineParameterNameHintsAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken)
+        {
+            var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var spans = new List<InlineParameterHint>();
```

Comment:
remove this and just do `var spans = ...` below.

Reply:
Fixed